src/main/resources/templates/cypher/form.html

```diff
@@ -8,42 +8,99 @@
+    <script src='https://api.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.js'></script>
+    <link href='https://api.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.css' rel='stylesheet' />
 </head>
 <body>
-<div class="container">
-    <h2><span th:text="${heading}">Nadpis</span></h2>
-    <form action="#" th:action="@{/cypher/save}" th:object="${cypher}" method="post">
-        <div class="table-responsive">
-            <table class="table" style="font-size: 18px">
-                <tr>
-                    <td>Id</td>
-                    <td><input th:field="*{cypherId}" readonly/></td>
-                </tr>
-                <tr>
-                    <td>Jméno</td>
-                    <td><input th:field="*{name}"/></td>
-                </tr>
-                <tr>
-                    <td>Pořadí</td>
-                    <td><input type="number" th:field="*{stage}"/></td>
-                </tr>
-                <tr>
-                    <td>Souřadnice</td>
-                    <td><input type="text" th:field="*{location}"/></td>
-                </tr>
-                <tr>
-                    <td>Odpověď</td>
-                    <td><input type="text" th:field="*{codeword}"/></td>
-                </tr>
-                <tr>
-                    <td></td>
-                    <td><input type="submit" value="Uložit"/>
-                    </td>
-                </tr>
-            </table>
-        </div>
-    </form>
-</div>
+    <div class="container">
+        <h2><span th:text="${heading}">Nadpis</span></h2>
+        <form action="#" th:action="@{/cypher/save}" th:object="${cypher}" method="post">
+            <div class="table-responsive">
+                <table class="table" style="font-size: 18px">
+                    <tr>
+                        <td>Id</td>
+                        <td><input th:field="*{cypherId}" readonly/></td>
+                    </tr>
+                    <tr>
+                        <td>Jméno</td>
+                        <td><input th:field="*{name}"/></td>
+                    </tr>
+                    <tr>
+                        <td>Pořadí</td>
+                        <td><input type="number" th:field="*{stage}"/></td>
+                    </tr>
+                    <tr>
+                        <td>Souřadnice</td>
+                        <td><input id="latLng" type="text" th:field="*{location}" readonly/></td>
+                    </tr>
+                    <tr>
+                        <td>Odpověď</td>
+                        <td><input type="text" th:field="*{codeword}"/></td>
+                    </tr>
+                    <tr>
+                        <td></td>
+                        <td><input type="submit" value="Uložit"/>
+                        </td>
+                    </tr>
+                </table>
+            </div>
+        </form>
+    </div>
+    <div id='map' style='width: 100%; height: 300px;'></div>
+    <script>
+        var map;
```

Comment:
Tip: Ty deklarace jde by měli jít  zjednodušit jako:

```
var map,
    activeMarker;
```

Když je jich tam víc, tak se to zpřehlední.

Reply:
Done